docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+2. Synchronous loading using the `FILES()` table function
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
+
+```SQL
+DROP TABLE IF EXISTS user_behavior;
+
+CREATE TABLE `user_behavior` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`) BUCKETS 1 
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+#### Gather connection details
+
+Loading data from S3 requires having the:
+
+- S3 endpoint
+- S3 region
+- Access key and secret
+
+#### Start a Broker Load
+
+This `LOAD` job has four main sections:
+
+- `LABEL`: A string used to query the state of a `LOAD` job.
+- `LOAD` declaration: The source URI, destination table, and the source data format.
+- `BROKER`: The connection details for the source.
+- `PROPERTIES`: Timeout value and any other properties to apply to this job.
+
+> Tip
+>
+> The dataset used in these examples is available in a StarRocks S3 bucket any valid `aws.s3.access_key` and `aws.s3.secret_key`. Substitute your credentials for `AAA` and `BBB` in the commands below.
+
+```SQL
+LOAD LABEL user_behavior
+(
+    DATA INFILE("s3a://starrocks-datasets/user_behavior_sample_data.parquet")
+    INTO TABLE user_behavior
+    FORMAT AS "parquet"
+ )
+ WITH BROKER
+ (
+    "aws.s3.enable_ssl" = "true",
+    "aws.s3.use_instance_profile" = "false",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/6e7b412a-bc6f-469f-9316-aa6ad57f6e59)
Shall we change these to "AAA" and "BBB"? 'cause the sentence "Substitute your credentials for `AAA` and `BBB` in the commands below." above the code snippet says "AAA" and "BBB".

Reply:
I need to look at the other doc, I didn't know there were more choices.

Reply:
added a note when I first introduce the topic and a link in More information.